package.json

```diff
@@ -148,7 +148,7 @@
     "babel-loader": "9.1.3",
     "babel-plugin-module-resolver": "5.0.0",
     "deep-freeze": "0.0.1",
-    "detox": "20.11.3",
```

Comment:
Do we need the `detox` dependency at the base package at all? I don't see any detox import in any file, but may be needed for other things I am not aware of.

Reply:
Yes, particularly when building androidTest (required for Detox testing), see:
- https://github.com/mattermost/mattermost-mobile/blob/main/android/build.gradle#L37
- https://github.com/mattermost/mattermost-mobile/blob/main/android/app/build.gradle#L148
- https://github.com/mattermost/mattermost-mobile/blob/main/android/app/src/androidTest/java/com/mattermost/rnbeta/DetoxTest.java